views/lib/forms/form-date-input.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="form-group">
+    {{#if label}}
+        <label for={{id}}>{{label}}</label>
+    {{/if}}
+    <div class="{{class}}">
+    <input
+            class="form-control"
+            data-datetime name={{id}}
+            autocomplete="off" id={{id}}
+            type="text"
+            value="{{value}}"
```

Comment:
value should match iso string, not formatted string (this one should be displayed only)